### PR TITLE
Standard User Registration

### LINKED ISSUE #4: Standard User Registration

### PR DESCRIPTION
**Pull request by vdevic01**
Closes #4

### CODE REVIEW

File: front/src/app/modules/layout/register-page/register-page.component.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          Passwords are not matching
+        </mat-error>
+      </mat-form-field>
+
+      <mat-error *ngIf="errorMessage !== ''">
+        {{errorMessage}}
+      </mat-error>
+      <button mat-raised-button color="primary" type="submit" (click)="submitForm()" [disabled]="registerForm.invalid">Sign Up</button>
+    </form>
+    <a class="action-link" href="/register">Already have an account? Login</a>
```
Comment (kaca01) [inline front/src/app/modules/layout/register-page/register-page.component.html:47]: Set path to login page

File: front/src/app/modules/layout/register-page/register-page.component.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        ReactiveFormsModule
+    ],
+  templateUrl: './register-page.component.html',
+  styleUrl: './register-page.component.css'
+})
+export class RegisterPageComponent {
+  errorMessage: string = "";
+  registerForm:FormGroup = new FormGroup({
+    email: new FormControl('', [Validators.required, Validators.email]),
+    username: new FormControl('', [Validators.required]),
```
Comment (kaca01) [inline front/src/app/modules/layout/register-page/register-page.component.ts:39]: max length validation?

**Comment by kaca01** [comment]
How am I supposed to change password?

**Comment by vdevic01** [comment]
> How am I supposed to change password?

Currently, password change is only used for super admin. The first time a super admin logs in, he/she is required to change the password. The same service may later be used to give other users a possibility to change the password but that's not a part of this feature.

File: front/src/app/modules/layout/register-page/register-page.component.html (hunk 1 above)
Comment (vdevic01) [reply front/src/app/modules/layout/register-page/register-page.component.html]: Fixed with most recent commits.

File: front/src/app/modules/layout/register-page/register-page.component.ts (hunk 2 above)
Comment (vdevic01) [reply front/src/app/modules/layout/register-page/register-page.component.ts]: Fixed with most recent commits.

Comment (kaca01) [reply front/src/app/modules/layout/register-page/register-page.component.ts]: Great! As far as I am concerned, you can merge.

**Pull request merged**